a4kSubtitles/services/subsource.py

```diff
@@ -70,11 +52,10 @@ def parse_search_response(core, service_name, meta, response):
     if "subs" not in results:
         return []
 
-    movie_details = results.get("movie", {})
+    # movie_details = results.get("movie", {})
```

Comment:
Prefer to remove commented out code from the repo. The changes can always be brought back from the history.